data/scene/earth/earth.mod

```diff
@@ -63,6 +58,67 @@ return {
         },
         GuiName = "/Solar/Planets/Earth"
     },
+    -- Earth Atmosphere
+    {
+         Name = "EarthAtmosphere",
+         Parent = "Earth",
+         Renderable = {
+              Type = "RenderableAtmosphere",
+              Atmosphere = {
+                Type = "RenderablePlanet",
+                -- Atmosphere radius in Km
+                AtmosphereRadius = 6447.0,
+                --PlanetRadius    = 6378.137,
+                PlanetRadius    = 6356.752,
+                --PlanetRadius    = 6360.0,
+                PlanetAverageGroundReflectance = 0.1,
+                Rayleigh = {
+                    Coefficients = {
+                        -- Wavelengths are given in 10^-9m
+                        Wavelengths = {680, 550, 440},
+                        -- Reflection coefficients are given in km^-1
+                        Scattering = {5.8E-3, 13.5E-3, 33.1E-3},
+                        -- In Rayleigh scattering, the coefficients of absorption and scattering are the same.
+                    },
+                    -- Thichkness of atmosphere if its density were uniform, in Km
```

Comment:
The thickness is expressed in Km to maintain dimensional homogeneity. Do you prefer meters instead (OS homogeneity)?